_posts/2023-03-07-vanilla-os-2.0-orchid-initial-work.md

```diff
@@ -0,0 +1,63 @@
+---
+title: "Vanilla OS 2.0 Orchid - Initial Work"
+description: "We are happy to announce Vanilla OS 2.0 Orchid! With significant changes that we'd like to go over, like changing the base to Debian Sid, OCI support in ABRoot and more."
+published: true
+layout: article
+comments: true
+---
+
+ 
+## Using Debian Sid as a Base
```

Comment:
Should we change this? I feel like it's a bit weird that it's worded that way.
```suggestion
## Moving to Debian Sid
```

Reply:
Sure, anything that works.